cartridge/auth-backend.lua

```diff
@@ -160,6 +161,11 @@ local function edit_user(username, password, fullname, email)
 
     if password ~= nil then
         user.password_data = update_password(password, user.password_data)
+        if user.version == nil then
+            user.version = 1
+        else
+            user.version = user.version + 1
+        end
```

Comment:
It would be nice to preserve the session which updates the password. Otherwise the next request will redirect to authorization page

Reply:
I start to refresh current user session. Add new test case.